docs/build-docs.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+rm -f ./_FIRST_RUN.flag
+
+ARCH=$(uname -m)
+export PATH="${CONDA}/bin:${PATH}"
+
+curl \
+    -L \
+    -o ${HOME}/conda.sh \
+    "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-${ARCH}.sh"
+
+/bin/bash ${HOME}/conda.sh -b -p ${CONDA}
+
+conda config --set always_yes yes --set changeps1 no
+
+mamba create \
+    --yes \
+    --name docs-env \
+    --file env.yml || exit -1
+
+source activate docs-env
+
+${CONDA}/envs/docs-env/bin/python \
```

Comment:
Why after `source activate docs-env` we can't simply use `python` and need to specify full path?

Reply:
ah this is left over from debugging. I ran into an issue similar to https://github.com/conda/conda/issues/3200 where `source activate` can fail when used in a bash script with `set  -e`.

This can be reverted to just `python`.

Reply:
updated in https://github.com/microsoft/LightGBM/pull/4957/commits/0c312eef03d1600853d4e35f8198ee79b6514cb0